scripts/provision.sh

```diff
@@ -311,6 +312,37 @@ install_frontend() {
   done
 }
 
+install_postgresql() {
+  #Check if externally hosted PostgreSQL is enabled
+  if [ "${PG_EXT_ENABLED:-false}" = "true" ]; then
+    echo "ERROR: --install-postgresql can not not be used if you are using"
+    echo "externally hosted PostgreSQL(RDS, Azure Database for PostgreSql etc)."
+    echo "Set PG_EXT_ENABLED=false to fix this error."
+    echo
+    return
+  fi
+
+  start_init
+  configure
+  start_datastore
```

Comment:
Similar to `start_builder` you will need to add:

```
    while [ ! -f /hab/svc/builder-datastore/config/pwfile ]
    do
      sleep 2
    done
    local pg_pass=$(cat /hab/svc/builder-datastore/config/pwfile)
    cat <<EOT > pg_pass.toml
[datastore]
password = "$pg_pass"
EOT
  hab config apply builder-api.default $(date +%s) pg_pass.toml
```
Probably want to extract that to its own function.